src/components/ec-file-dropzone/ec-file-dropzone.vue

```diff
@@ -119,14 +132,36 @@ export default {
 
   &__img {
     @apply tw-fill-key-4 tw-mb-8;
+
+    &--is-disabled {
+      @apply tw-fill-gray-6;
+    }
+  }
+
+  &__title {
+    @apply tw-h3;
+    @apply tw-m-0;
+
+    &--is-disabled {
+      @apply tw-text-gray-5;
+    }
   }
 
   &__subtitle {
-    @apply tw-small-text tw-cursor-pointer;
+    @apply tw-small-text;
+
+    &--is-disabled {
+      @apply tw-text-gray-6;
+    }
   }
 
   &--dragging {
     @apply tw-border-key-4;
```

Comment:
The blue border is being displayed on dragging even when the component is disabled. I think that we should not apply this effect when we are in disabled status.